css/style.css

```diff
@@ -0,0 +1,75 @@
+html {
+  font-size: 100%;
+  overflow-y: scroll;
+}
+
+body {
+  margin-left: auto;
+  margin-right: auto;
+  font-size: 16px;
+  line-height: 1.61;
+  font-weight: 400;
+  width: 1024px;
+  font-family: 'Fantasy', fantasy;
+  color: #333;
```

Comment:
on 14 line color value contains 3 symbols (#333), and on the next line color value contains 6 symbols (#ffffff)  - it's recommended to follow a unified approach